types/vue-scrollto/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for vue-scrollto 2.7
+// Project: https://github.com/rigor789/vue-scrollto#readme
+// Definitions by: Kovács Vince <https://github.com/vincekovacs>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { PluginFunction } from "vue";
+
+declare namespace VueScrollTo {
+    interface Options {
+        // The element you want to scroll to.
```

Comment:
I believe the `/**/` comment is better, it can be visible in IDEs
```js
/**
 * The element you want to scroll to.
 */
```

Reply:
I mainly use `/** ... */` style comments for method or function declarations. My team opinion is that for short information single line comment is better than block comment. We using WebStorm and Atom for developing but in booth IDE the block level and the single line comments are visible enough. Maybe better visibility can get with one empty line before each property:

```javascript
// The element you want to scroll to.
el?: string;
element?: string;

// The container that has to be scrolled. Default: body
 container?: string;
```